pkg/sysfs/system.go

```diff
@@ -962,26 +964,28 @@ func (sys *system) discoverPackages() error {
 }
 
 func (sys *system) discoverSst() error {
-	if !SstSupported() {
+	if !sst.SstSupported() {
 		sys.Info("Speed Select Technology (SST) support not detected")
 		return nil
 	}
 
 	for _, pkg := range sys.packages {
-		sstInfo, err := getSstPackageInfo(pkg)
+		sstInfo, err := sst.GetPackageInfo(pkg.id)
 		if err != nil {
 			return fmt.Errorf("failed to get SST info for package %d: %v", pkg.id, err)
 		}
 		sys.DebugBlock("", "Speed Select Technology info detected for package %d:\n%s", pkg.id, utils.DumpJSON(sstInfo))
 
 		if sstInfo.CPEnabled {
 			ids := pkg.cpus.SortedMembers()
-			clos, err := getCPUClosIDs(ids)
-			if err != nil {
-				return fmt.Errorf("failed to get SST-CP clos ids for package %d: %v", pkg.id, err)
-			}
-			for i, id := range ids {
-				sys.cpus[id].sstClos = clos[i]
+
+			for _, id := range ids {
+				clos, err := sst.GetCPUClosID(id)
+				if err != nil {
+					return fmt.Errorf("failed to get SST-CP clos ids for package %d: %v", pkg.id, err)
+				}
+
+				sys.cpus[id].sstClos = clos
```

Comment:
@klihub / @marquiz can you check that this looks ok to you, the Clos get function was changed when code was moved to goresctrl so I had to change this part a bit more

Reply:
Looks good 👍